02 - Community Expansions/02.05 All Community Expansions/Themes/Minimal.md

```diff
@@ -132,7 +138,7 @@ Modes: [[Dark-mode themes|dark]], [[Light-mode themes|light]]
 - [[calendar|Calendar]]
 - [[obsidian-checklist-plugin|Checklist]]
 - [[obsidian-codemirror-options|CodeMirror Options]]
-- [[obsidian-dictionary|obsidian-dictionary]]
```

Comment:
Was this a rename? Is the old note removed?

Reply:
This change fixes a dangling/broken link - with discussion starting here: https://discord.com/channels/686053708261228577/891399239198982195/910289335708958720

An author had updated an ID in a plugin, but not in the corresponding theme...

So this change converted this broken link:

```
- [[obsidian-dictionary|obsidian-dictionary]]
```

to this working one:

```
- [[obsidian-dictionary-plugin|Dictionary]]
```